django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -108,6 +128,8 @@
         win.close();
     }
 
+    let popupIndex = 0;
+
```

Comment:
I think this should live at the top ≈ln6.

Reply:
Will make the change. Also could you help with the linter failing. isort doesnt fail while running the pre-commit hook. What am I missing?

Reply:
I'll have a look.